content/_code-samples/airgapped-wallet/py/airgapped-wallet-TEST.py

```diff
@@ -33,7 +33,7 @@ def create_wallet(silent: False):
         pub, priv = keypairs.derive_keypair(seed)
         address = keypairs.derive_classic_address(pub)
  
-    return address, seed
```

Comment:
I just made a conflicting change (to lines 132 and 192 which call this function) in #1636 to fix this same problem with the tests. Maybe check that you're branched off the latest version so this doesn't break anything or conflict?